c++/src/kj/table.h

```diff
@@ -893,7 +893,10 @@ inline size_t probeHash(const kj::Array<HashBucket>& buckets, size_t i) {
 
 kj::Array<HashBucket> rehash(kj::ArrayPtr<const HashBucket> oldBuckets, size_t targetSize);
 
-uint chooseBucket(uint hash, uint count);
+inline uint chooseBucket(uint hash, uint count) {
+  KJ_IASSERT(__builtin_popcount(count) == 1, "hash bucket count must be power of two!");
```

Comment:
MSVC wants __popcount() here – thankfully we have already handled that in `popCount()` defined in kj/common.h